src-cpp/app/modules/main/view.cpp

```diff
@@ -29,25 +24,25 @@ void View::load()
 
 void View::addItem(Shared::Models::SectionItem* item)
 {
-    m_sectionModelPtr->addItem(item);
-    emit sectionsModelChanged();
+    m_sectionModel.addItem(item);
+    // emit sectionsModelChanged(); // FIXME: that's wrong, sectionModel* property didn't change
```

Comment:
It would be only valid in such case:

`Q_PROPERTY(Shared::Models::SectionModel* sectionsModel READ getSectionsModel NOTIFY sectionsModelChanged)`
```cpp
void changeModel(std::unique_ptr<Shared::Models::SectionModel> newModel)
{
    m_sectionModel = std::move(newModel);
    emit sectionsModelChanged();
}
```